jobs/pull-kubernetes-e2e-gce-cri.sh

```diff
@@ -58,7 +58,7 @@ export NUM_NODES="3"
 export KUBE_NODE_OS_DISTRIBUTION="gci"
 
 # Enable experimental CRI integration
-export KUBELET_TEST_ARGS="--experimental-cri=true"
+export KUBELET_TEST_ARGS="--enable-cri=false"
```

Comment:
Reusing projects is fine, but I do suggest at least rename this pull job to be non-cri, since it's not in testgrid and will show up in PR presubmits and will be misleading.